pom.xml

```diff
@@ -120,6 +120,11 @@
                             </dependency>
                         </dependencies>
                         <configuration>
+                            <archive>
+                                <manifest>
+                                    <addClasspath>true</addClasspath>
+                                </manifest>
+                            </archive>
```

Comment:
What is this for?

Reply:
Hello, sorry i forgot it, I am using it on windows to be able to run maven clean install and other commands, otherwise it throws error: CreateProcess error=206, The filename or extension is too long, i can delete it if needed

Reply:
Is this for the `maven-surefire-plugin` and `maven-failsafe-plugin`? If so, then, this fix should be applied to the `strongbox-parent` project instead.

Reply:
Removed